source/configuration_manual/authentication/oauth2.rst

```diff
@@ -209,6 +209,12 @@ Currently Dovecot oauth2 library implements the following features of JWT tokens
 * AUD support (this is checked against scope, if provided)
 * AZP support
 
+.. versionchanged:: v2.3.21
+
+  AUD check now checks client_id, not scope.
```

Comment:
Put all these into the same paragraph together with versionchanged, here and below.

Reply:
afaik sphinx treats consecutive lines as same paragraph if there is no empty line between.

Reply:
Yeah, I mean I don't like these standalone "Changed in version v2.3.21" paragraphs, since it's not clear whether it refers to the paragraph above or below or something else (before I changed the existing ones and started complaining about these, they were kind of 50-50). So `.. versionchanged:: v2.3.21 AUD check now...` makes it much clearer.